README.md

```diff
@@ -1 +1,288 @@
-# Hello-Microverse-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```

Comment:
- [ ] This readme file is a template that you should have modified to match your project and remove sections that don't apply. Examples of sections that don't apply include demo and test.